chapter_linear-networks/softmax-regression_vn.md

```diff
@@ -36,7 +37,10 @@ In practice, we are more often interested in classification: asking not *how muc
 * Which movie is Aston most likely to watch next?
 -->
 
-*dịch đoạn phía trên*
+* Email này thuộc thư mục thư rác hay hộp thư đến?
+* Khách hàng này nhiều khả năng *đăng ký* hay *không đăng ký* một dịch vụ thuê bao?
```

Comment:
Chỗ này bài toán đang nhắc tới là phân loại "đăng ký / không đăng ký", mình dịch thẳng ra thành "Xác suất ..." sẽ bị nhầm thành bài toán hồi quy rồi.
```suggestion
* Khách hàng này nhiều khả năng sẽ *đăng ký* hay *không đăng ký* dịch vụ thuê bao hơn?
```

Reply:
@lkhphuc không nhầm đâu. Nó là bài toán phân loại đấy. Xem line 46 nhé:

```mardown
Colloquially, machine learning practitioners overload the word *classification* to describe two subtly different problems:
(i) those where we are interested only in *hard* assignments of examples to categories; and (ii) those where we wish to make *soft assignments*,
i.e., to assess the *probability* that each category applies.
The distinction tends to get blurred, in part, because often, even when we only care about hard assignments, we still use models that make soft assignments.
```

Reply:
Mình hiểu, nhưng chỗ này vốn đang đưa ra các bài toán ví dụ cho việc phân loại, nên mình vẫn thấy nên giữ ý nghĩa đấy.

Reply:
Nó vẫn là một ví dụ bài toán phân loại. Và khi đọc tới đoạn sau sẽ rõ hơn là bài toán phân loại có 2 biến thể.

Reply:
@tanduong chỗ này theo mình không nên tự suy luận mà nên dịch theo ý tác giả. Các bác ấy chỉ nói khả năng cao chọn cái này hay cái kia chứ không nói về xác suất. Mình thấy để nguyên như @honghanhh là ok.

Reply:
e sẽ giữ nguyên phần dịch này nhé a @tiepvupsu  @tanduong